packages/cfd/src/Containers/cfd-reset-password-modal.tsx

```diff
@@ -22,8 +21,11 @@ const ResetPasswordIntent = ({ current_list, context, children, is_eu, ...props
     let group, type, login, title, server;
     if (has_intent && current_list) {
         [server, group, type] = reset_password_intent.split('.');
+        const title_type = type === 'financial_demo' ? 'financial' : type;
```

Comment:
@hirad-deriv how we handle the other demos? since there is no demo in non of the keys of `getMtCompanies` return? for example `synthetic`

Reply:
This case only happens for MF accounts, we do not have financial_demo in our list and that was the reason that we were receiving error. For CR accounts which have synthetic accounts, `financial_svg` is returned to us and that is the reason everything works fine for CR accounts.